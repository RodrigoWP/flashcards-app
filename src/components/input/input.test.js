import React from 'react'
import { shallow } from 'enzyme'
import { Input } from '.'

it('Should be render correctly', () => {
    const Component = shallow(
        <Input label='label' />
    )

    expect(Component).toMatchSnapshot()
})

