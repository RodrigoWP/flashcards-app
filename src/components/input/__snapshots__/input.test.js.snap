// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should be render correctly 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#3b495e",
        "fontSize": 12,
        "fontWeight": "bold",
        "marginLeft": 5,
      }
    }
  >
    LABEL
  </Text>
  <TextInput
    allowFontScaling={true}
    style={
      Object {
        "color": "#3b495e",
        "fontSize": 14,
        "height": 40,
        "paddingLeft": 5,
      }
    }
    underlineColorAndroid="#3b495e"
  />
</View>
`;
